docs/reference/admin/layouts/directives.mdx

```diff
@@ -9,8 +9,16 @@ import _indexSource from '!!raw-loader!./_Example/index.tsx'
 
 # <span className="version">Interface 1.2+</span> Directives
 
-Directives are React components that can be used to pass values (even rendered) to upper context
-and control your app from within the page—**to direct it what to do**, hence the name.
+**Use directives to to pass values to upper contexts using JSX components.**
+
+Directives are React components that can be used to pass values to upper contexts
+just by rendering `<Directive>` components and specifying the name and content.
+
+Similar to slots that give you a way to escape from the tied page layout, directives give you a way to escape
+from the tied page behavior.
+
+In most cases you can use it to control your app from within the page to **direct it what to do** (hence the name),
+e.g. when you need some pages to stretch to the full width of the screen.
```

Comment:
I would reorder this page.. i think the structure should be like this:
- intro (until this line)
- then there should be "creating directives" section, because this is the first step you must do. I would omit the `FullWidth` from the main code example, because it is optional (maybe just keep the "tip")
- then there should be a "setting directive values", but with just one value; multiple <Directives> just complicates things
- then "accessing directive values" usection
- and last caveats like the order of evaluation

Reply:
Done